test/cnvkit-tools/genome2access.cwl

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: "cwl:draft-3.dev2"
```

Comment:
I am not sure what does it mean, that's what's left from the old template, it will be replaced anyway when draft-4 comes